### PR TITLE
Feat: Added List component

### PR DESCRIPTION
**Pull request by kiranpratyush**
updated index.html according to the documentation site(though some changes are pending for header ignore it for now updated the style.css
Below is the live working link please review it and give help full feedback

 Components Defined - 
- List

Live Link -  : [Testing](https://testing2311.netlify.app/List/)

### CODE REVIEW

**Review by bipin7yadav — APPROVED** [review]

File: List/index.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                <li>Green tea</li>
+              </ul>
+            </li>
+            <li>Milk</li>
+          </ul>
+        </div>
+        <div
+          class="container"
+          style="
+            height: 350px;
```
Comment (mimansha-swarup) [inline List/index.html]: try not to use inline  styling

File: List/style.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.list {
+  list-style-type: None;
+  background-color: white;
+  padding: 30px 20px;
```
Comment (mimansha-swarup) [inline List/style.css]: try using relative units

Comment (kiranpratyush) [reply List/style.css]: I have updated the units and removed inline styling can you please approve it

File: List/index.html (hunk 1 above)
Comment (kiranpratyush) [reply List/index.html]: > try not to use inline styling

I have  changed accordingly can you please approve it

**Pull request merged**